README.md

```diff
@@ -17,8 +17,9 @@ SQL DSLs by:
 - Supporting mandatory and optional named parameters.
 - Allowing query composability with ease.
 - Working out of the box with PostgreSQL using
-  [Ecto](https://github.com/elixir-ecto/ecto_sql) or
-  [Postgrex](https://github.com/elixir-ecto/postgrex).
+  [Ecto](https://github.com/elixir-ecto/ecto_sql),
+  [Postgrex](https://github.com/elixir-ecto/postgrex) or
+  [MyXQL](https://github.com/elixir-ecto/myxql).
```

Comment:
AyeSQL works out-of-the-box with Ecto and Posgrex, but not with MyXQL. Let me elaborate: AyeSQL provides two query runners:

- `AyeSQL.Runner.Ecto` (default).
- `AyeSQL.Runner.Postgrex`.

If you want to query PostgreSQL without Ecto, you can do it by doing something like the following:

```elixir
import AyeSQL, only: [defqueries: 3]

defqueries(MyQueries, "sql/queries.ex", runner: AyeSQL.Runner.Postgrex)
```

And then call the queries as follows:

```elixir
iex> MyQueries.get_user_by_name([name: "Josep"], conn: conn)
```

MyXQL works through Ecto, but it won't work out-of-the-box without it. That why saying _MyXQL works out-of-the-box with AyeSQL_ would be a lie. MyXQL works because Ecto works out-of-the-box. If Ecto drops support of MyXQL, we would need to implement an AyeSQL runner in order to use it.

Reply:
I would be happy to do that when I have more free time 🤗